README.md

```diff
@@ -329,12 +329,12 @@ mamba activate "$ENV_NAME"
 # install cudf packages built in CI and test them in newly created environment...
 ```
 
-The `--file_key` argument is passed the `test` key name from the `files` configuration. Additional flags are used to generate a single dependency file. When the CLI is used in this fashion, it will print to `stdout` instead of writing the resulting contents to the filesystem.
+The `--file-key` argument is passed the `test` key name from the `files` configuration. Additional flags are used to generate a single dependency file. When the CLI is used in this fashion, it will print to `stdout` instead of writing the resulting contents to the filesystem.
 
-The `--file_key`, `--output`, and `--matrix` flags must be used together. `--matrix` may be an empty string if the file that should be generated does not depend on any specific matrix variations.
+The `--file-key`, `--output`, and `--matrix` flags must be used together. `--matrix` may be an empty string if the file that should be generated does not depend on any specific matrix variations.
 
-The `--prepend-channels` argument accepts a semi-colon separated list of additional channels to use, like `rapids-dependency-file-generator --prepend-channels "my_channel;my_other_channel"`.
-If both `--output` and `--prepend-channels` are provided, the output format must be conda.
+The `--prepend-channel` argument accepts additional channels to use, like `rapids-dependency-file-generator --prepend-channel "my_channel" --prepend-channel "my_other_channel"`.
```

Comment:
We probably don't need to quote this, now that the semicolon is not in the string. I'm fine either way if you like the quotes.
```suggestion
The `--prepend-channel` argument accepts additional channels to use, like `rapids-dependency-file-generator --prepend-channel my_channel --prepend-channel my_other_channel`.
```

Reply:
Removed.